src/functions.js

```diff
@@ -365,6 +364,28 @@ defineFunction(["\\mathop"], {
     };
 });
 
+defineFunction(["\\limits"], {
```

Comment:
I decided to implement limits (and nolimits) so that they could be used elsewhere.  According to [latex.ltx](https://www.tug.org/svn/texlive/trunk/Master/texmf-dist/tex/latex/base/latex.ltx?view=co) it looks `\mathop` defaults to enabling limits.  Maybe we should do the same.